modules/standard/HDFS.chpl

```diff
@@ -18,19 +18,315 @@
  */
 
 /*
-   HDFS module
+This module implements support for the 
+`Hadoop <http://hadoop.apache.org/>`_ 
+`Distributed Filesystem <http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html>`_ (HDFS).
```

Comment:
I'm not sure what you mean. It already says Hadoop Distributed Filesystem (HDFS)